src/Logary.Prometheus/Exporter.fs

```diff
@@ -0,0 +1,171 @@
+﻿namespace Logary.Prometheus
+open System
+open System.Text
+open System.Text.RegularExpressions
+open Logary.Metric
+open Suave
+open Suave.Filters
+open Suave.Operators
+open Suave.Writers
+open Hopac
+
+
+type ExporterConf =
+  {
+    metricNameTransformer: string -> string
+    metricRegistry: MetricRegistry
+    webConfig: SuaveConfig
```

Comment:
It feels a bit like this should be under a namespace `Logary.Prometheus.Exporter` or something that relates to the polling, so that the first types in the `.Prometheus` namespace aren't polled-exporters. Thinking that we might want to feed prometheus with push at some point, e.g. when we would like to ingest historical data from Logary.

Reply:
👍